products/ionic.md

```diff
@@ -13,6 +13,10 @@ eolColumn: Maintenance
 releaseDateColumn: true
 extendedSupportColumn: true
 
+identifiers:
+-   purl: pkg:github/ionic-team/ionic
```

Comment:
```suggestion
-   purl: pkg:github/ionic-team/ionic-framework
```

https://github.com/ionic-team/ionic redirects to https://github.com/ionic-team/ionic-framework.